src/StoreContext/StoreContext.tsx

```diff
@@ -508,8 +515,10 @@ export const StoreProvider = ({
           payload: {
             ...(code ? { code } : {}),
             ...(editorPosition ? { editorPosition } : {}),
-            ...(editorHeight ? { editorHeight } : {}),
-            ...(editorWidth ? { editorWidth } : {}),
+            ...(editorHeightPercentage
+              ? { editorHeightPercentage }
+              : undefined),
+            ...(editorWidthPercentage ? { editorWidthPercentage } : undefined),
```

Comment:
Consistency nitpick 😄 

```suggestion
            ...(editorHeightPercentage ? { editorHeightPercentage } : {}),
            ...(editorWidthPercentage ? { editorWidthPercentage } : {}),
```